hydroshare/irods/ruleGenerateBagIt_HS.r

```diff
@@ -74,5 +74,5 @@ generateBagIt {
   ### - writes to rodsLog
   msiWriteRodsLog("BagIt bag files created in place: *BAGITDATA <- *BAGITDATA",*Status);
 }
-INPUT *BAGITDATA="/dummy/dummy/dummy" 
+INPUT *BAGITDATA="/hydrotest41Zone/home/hyi", *DESTRESC="demoResc"
```

Comment:
This is just a dummy placeholder, and will be overriden by parameters being passed in at run time, but I will change it to be "dummy" to make it clear.